tests/MongoDB.Bson.Tests/Serialization/Serializers/BsonPrimitiveSerializerTests.cs

```diff
@@ -72,6 +84,16 @@ public void TestTrue()
             var rehydrated = BsonSerializer.Deserialize<TestClass>(bson);
             Assert.True(bson.SequenceEqual(rehydrated.ToBson()));
         }
+
+        [Fact]
+        public void TestPrivateFieldWithBsonRepresentation()
+        {
+            var testValue = true;
+            var serializedDocument = BsonDocument.Parse($"{{ _b : '{testValue}' }}");
+
+            var deserialized = BsonSerializer.Deserialize<TestClassWithPrivate>(serializedDocument);
+            Assert.Equal(testValue, deserialized.GetPrivateB());
```

Comment:
Can be slightly simplified:

    var testValue = true;
    var json = $"{{ _b : '{testValue}' }}";

    var deserialized = BsonSerializer.Deserialize<TestClassWithPrivate>(json);
    Assert.Equal(testValue, deserialized.GetPrivateB());

Same for similar tests  below.

Reply:
done